dart/lib/src/sentry_traces_sampler.dart

```diff
@@ -14,7 +14,12 @@ class SentryTracesSampler {
   SentryTracesSampler(
     this._options, {
     Random? random,
-  }) : _random = random ?? Random();
+  }) : _random = random ?? Random() {
+    if (_options.tracesSampler != null && _options.tracesSampleRate != null) {
+      _options.logger(SentryLevel.warning,
+          'Both tracesSampler and sampleRate are set - only tracesSampler will be used.');
```

Comment:
@buenaflor i mean changing this log that you just added with something like:
`Both tracesSampler and sampleRate are set. TracesSampler will take precedence and fallback to sampleRate when it returns null.`

Reply:
I see, makes sense